src/map/atcommand.c

```diff
@@ -8366,6 +8366,36 @@ void atcommand_commands_sub(struct map_session_data* sd, const int fd, AtCommand
 	dbi_destroy(iter);
 	clif->message(fd,line_buff);
 
+	if (atcommand->binding_count) {
+		int i, count_bind = 0;
+		int gm_lvl = pc_get_group_level(sd);
+		size_t slen;
+		for (i = 0; i < atcommand->binding_count; i++) {
+			if (gm_lvl >= ((type == COMMAND_ATCOMMAND) ? atcommand->binding[i]->group_lv : atcommand->binding[i]->group_lv_char)) {
+				slen = strlen(atcommand->binding[i]->command);
```

Comment:
can you move the `size_t slen` variable declaration here? Right now it's outside the `for` for no reason